.github/workflows/integration-tests.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           path: ~/mne_data
       - run: ./tools/github_actions_download.sh
         working-directory: mne-python
-      - run: pytest mne/viz/_brain mne/viz/tests/test_3d.py mne/viz/backends
+      - run: pytest mne/viz/_brain mne/viz/tests/test_3d.py mne/viz/backends -W ignore::pyvista.core.errors.PyVistaDeprecationWarning
```

Comment:
This may not be relevant to this discussion, but what are the reasons for integration testing other repositories? In the past, there have been cases where CI has been stopped due to errors that occurred in repositories other than PyVista. What is the necessity of running CI here?

Reply:
These integration tests were added in https://github.com/pyvista/pyvista/pull/3967 and spun out of https://github.com/pyvista/pyvista/pull/3385. 

We are in a scenario where downstream projects heavily depend on PyVista, and these integration tests play a crucial role in ensuring that modifications to PyVista do not inadvertently break or disrupt these projects. 

By conducting integration tests here in PyVista's CI, we can assess how well the library integrates with external software and identify potential issues that may arise due to changes before the issues arise. This approach provides a safety net, allowing us to catch and rectify any compatibility problems early on -- thereby reducing "release pain". By validating that our changes do not break downstream projects through CI tests here, we can instill confidence in the stability and reliability of PyVista, fostering a smooth development process and minimizing the risk of disruptions in downstream projects that may ultimately have more users than PyVista directly. 

Ultimately, integration testing serves as a vital measure to safeguard the overall integrity and functionality of what we are shipping